### PR TITLE
[JENKINS-68642] Agents log link is shown to users without permission

### PR DESCRIPTION
**Pull request by Langer0416**
See [JENKINS-68642](https://issues.jenkins.io/browse/JENKINS-68642).

This screenshot is to clarify the code.
Access denied

### Proposed changelog entries

User without permission(Agent/Configire or Agent/Connect) could not view the "Node Log Detail".

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since="TODO")` or `@Deprecated(since="TODO", forRemoval=true)` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6734"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/slaves/ComputerLauncher/main.jelly
Comment (daniel-beck) [inline core/src/main/resources/hudson/slaves/ComputerLauncher/main.jelly:33]:
```suggestion
          <j:if test="${it.hasAnyPermission(it.CONFIGURE, it.CONNECT)}">
            <a href="log">${%See log for more details}</a>
```

**Review by daniel-beck — APPROVED** [review]
Looks reasonable. Untested.

File: core/src/main/resources/hudson/slaves/ComputerLauncher/main.jelly
Comment (Langer0416) [reply core/src/main/resources/hudson/slaves/ComputerLauncher/main.jelly:33]: @daniel-beck , I didn't change it to suggested code, since it could not work.

Comment (daniel-beck) [reply core/src/main/resources/hudson/slaves/ComputerLauncher/main.jelly:33]: Thanks for checking 👍

**Review by NotMyFault — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
